File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/simpleExample/SimpleExampleQuestViewHolder.java
package ru.nekit.android.qls.quest.view.mediator.types.simpleExample;

import android.content.Context;
import android.support.annotation.NonNull;
import android.view.View;
import android.widget.EditText;
import android.widget.TextView;

import ru.nekit.android.qls.R;
import ru.nekit.android.qls.utils.ViewHolder;

public class SimpleExampleQuestViewHolder extends ViewHolder {

    public EditText alternativeAnswerInput;
    TextView rightSideView;
    TextView leftSideView;

    SimpleExampleQuestViewHolder(@NonNull Context context) {
        super(context, R.layout.ql_simple_example);
        alternativeAnswerInput = (EditText) view.findViewById(R.id.alternative_answer_input);
        leftSideView = (TextView) view.findViewById(R.id.tv_left_side);
        rightSideView = (TextView) view.findViewById(R.id.right_side_view);
    }

    @NonNull
    public View getView() {
        return view;
    }
}

File: questPresentation/src/main/java/ru/nekit/android/qls/quest/view/mediator/types/simpleExample/SimpleExampleQuestViewHolder.kt
package ru.nekit.android.qls.quest.view.mediator.types.simpleExample

import android.content.Context
import android.view.View
import android.widget.EditText
import android.widget.TextView

import ru.nekit.android.qls.R
import ru.nekit.android.utils.ViewHolder

//ver 1.0
class SimpleExampleQuestViewHolder internal constructor(context: Context) : ViewHolder(context, R.layout.ql_simple_example) {

    var alternativeAnswerInput: EditText = view.findViewById<View>(R.id.alternative_answer_input) as EditText
    var rightSideView: TextView = view.findViewById<View>(R.id.right_side_view) as TextView
    var leftSideView: TextView = view.findViewById<View>(R.id.tv_left_side) as TextView

}
